code/__DEFINES/input_list_options.dm

```diff
@@ -0,0 +1,10 @@
+
+/**
+ * Defines for tgui_input_list options.
+ * Use to avoid some poor sap putting in "rename" instead of "Rename".
+*/
+
+/// Used primarily in /datum/element/tool_renaming
+#define OPTION_RENAME "Rename"
+#define OPTION_DESCRIPTION "Description"
+#define OPTION_RESET "Reset"
```

Comment:
these don't need to be global defines, move them to the `tool_renaming.dm` file and then `#undef` them at the very end.